src/DevLicenseCore.sol

```diff
@@ -324,7 +324,24 @@ contract DevLicenseCore is Initializable, AccessControlUpgradeable, IDevLicenseD
 
         uint256 timestampInit = $._signers[tokenId][signer];
         uint256 timestampCurrent = block.timestamp;
-        isSigner_ = timestampCurrent - timestampInit <= $.periodValidity;
+        isSigner_ = (timestampInit != 0) && (timestampCurrent - timestampInit <= $.periodValidity);
```

Comment:
note: This subtraction is safe!

Reply:
Are you saying the previous version without the `(timestampInit != 0)` was safe?

Reply:
No, just that any subtraction of `uints` is always dodgy-looking.